operators/multiclusterobservability/controllers/multiclusterobservability/analytics/rightsizing_namespace_policy.go

```diff
@@ -0,0 +1,137 @@
+// Copyright (c) Red Hat, Inc.
+// Copyright Contributors to the Open Cluster Management project
+// Licensed under the Apache License 2.0
+
+package analytics
+
+import (
+	"context"
+	"encoding/json"
+
+	"github.com/cloudflare/cfssl/log"
+	monitoringv1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	configpolicyv1 "open-cluster-management.io/config-policy-controller/api/v1"
+	policyv1 "open-cluster-management.io/governance-policy-propagator/api/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// Helps in creating or updating existing Policy for the PrometheusRule
+func createOrUpdatePrometheusRulePolicy(
+	ctx context.Context,
+	c client.Client,
+	prometheusRule monitoringv1.PrometheusRule) error {
+
+	policy := &policyv1.Policy{
+		TypeMeta: metav1.TypeMeta{
+			APIVersion: "policy.open-cluster-management.io/v1",
+			Kind:       "Policy",
+		},
```

Comment:
Is this type metadata required? I guess it should not be required.

Reply:
True, removed type metadata.